lib/context_request_middleware/active_support/extract.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# rubocop:disable all
+class Array
```

Comment:
Isn't this the same as `Array.delete_if/Array.keep_if`?
Same again please remove the Monkeypatch.

Reply:
Monkeypatch removed.